cmd/notary/main.go

```diff
@@ -86,11 +86,12 @@ func init() {
 		fatalf("could not create Certificate X509FileStore: %v", err)
 	}
 
-	privKeyStore, err = trustmanager.NewKeyFileStore(finalPrivDir)
+	//TODO(mccauley): Appears unused? Remove it? Or is it here for early failure?
+	privKeyStore, err = trustmanager.NewKeyFileStore(finalPrivDir,
+		func(string, string, bool, int) (string, bool, error) { return "", false, nil })
```

Comment:
tried removing and just found that its used by keys.go. I'll remove the comment.